src/Data/Nat/Binary/Minus.agda

```diff
@@ -0,0 +1,378 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Subtraction on Bin and some of its property proofs.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Nat.Binary.Minus where
+
+import Algebra.FunctionProperties as FuncProp
+open import Data.Empty using (⊥)
+open import Data.Fin using () renaming (zero to 0F; suc to sucF)
+open import Data.Nat as ℕ using (ℕ)
```

Comment:
Instead of renaming these and writing `0F`, `sucF 0F` and `sucF (sucF 0F)`,
you could use [`#_`](https://agda.github.io/agda-stdlib/Data.Fin.html#730): all your cases are concrete so Agda should see that
you can indeed embed `0`, `1`, and `2` into `Fin 3`.